hig.js

```diff
@@ -1,8 +1,10 @@
 var port = process.env.PORT || 3000,
     http = require('http'),
     fs = require('fs');
+var os = require('os');
+os = os.platform();
```

Comment:
this could be done in one line "var os =  require('os').platform();" Otherwise it is a little weird to read.

Reply:
Thanks! Ill change it. Makes sense.

Reply:
I made this change.